hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/query/matcher/GraterThanMatcher.java

```diff
@@ -0,0 +1,41 @@
+package pl.allegro.tech.hermes.management.infrastructure.query.matcher;
+
+import org.apache.commons.jxpath.JXPathException;
+import pl.allegro.tech.hermes.management.infrastructure.query.graph.ObjectGraph;
+
+public class GraterThanMatcher implements Matcher {
```

Comment:
That's duplicated with `LowerThanMatcher`. How about creating a `ComparisonMatcher` which would receive an operator function of (double, double) => boolean as parameter? Then implementing `GreaterThanMatcher` and `LowerThanMatcher` would just mean configuring it in a proper way, e.g.:
* greater: new ComparisonMatcher(path, parser.parseValue(node), (a, b) -> a > b);
* lower: new ComparisonMatcher(path, parser.parseValue(node), (a, b) -> a < b);